.github/src/EngineManager.java

```diff
@@ -20,77 +20,30 @@
 
 public class EngineManager implements ComputeAPI {
 
-    DataAPI dataStoreAPI;
-    EngineAPI computeEngineAPI;
-    EngineComputeRequest engineRequest;
-    /* j commented out 2/22 these instance variables only need to access the method calls in their
-    respective interfaces to transfer arguments to correct files
-    
-    public EngineManager(DataStorage dataStorage, EngineCompute engineCompute ) {
+    DataAPI ds;
+	EngineAPI ec;
+	
+	public EngineManager(DataAPI ds, EngineAPI ec) {
+		this.ds = ds;
+		this.ec = ec;
+	}
+
+	@Override
+	public ComputeResult compute(ComputeRequest request) {
+		Iterable<Integer> integers = ds.read(request.getInputConfig());
+		
+		//compute GCD
+		int gcd = ec.getGCD((List<Integer>) integers);
```

Comment:
I think there is a way to handle this in our engineAPI compute method but for now, this is okay.